Components/ViewComponents/IconButton/IconButtonComponentView.swift

```diff
@@ -0,0 +1,91 @@
+import UIKit
+
+protocol IconButtonComponentViewDelegate: class {
+    func iconButtonComponentView(_ iconButtonComponentView: IconButtonComponentView, didTapButtonFor component: IconButtonComponent)
+}
+
+public class IconButtonComponentView: UIView {
+
+    // MARK: - Internal properties
+
+    private let imageHeight: CGFloat = 20
+    private let imageWidth: CGFloat = 20
+
+    private let highlightedColor = UIColor(red: 0 / 255, green: 79 / 255, blue: 201 / 255, alpha: 1.0) // #004fc9
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.isAccessibilityElement = true
+        label.textColor = .primaryBlue
```

Comment:
It's fine that the color is here but we need to abstract all these colors into a theming thingie later.